rsts/deployment/cluster_config/general.rst

```diff
@@ -75,16 +75,15 @@ These are free-form key-value pairs used when filling the templates that the adm
 
 The keys represent templatized variables in `cluster resource template YAML <https://github.com/flyteorg/flyte/blob/1e3d515550cb338c2edb3919d79c6fa1f0da5a19/charts/flyte-core/values.yaml#L737,L760>`__ and the values are what you want to see filled in.
 
-
 In the absence of custom override values, ``templateData`` from the `FlyteAdmin config <https://github.com/flyteorg/flyte/blob/1e3d515550cb338c2edb3919d79c6fa1f0da5a19/charts/flyte-core/values.yaml#L719,L734>`__
-is used as a default. Note, these defaults are specified by domain but will be applied to every project-domain namespace combination.
+is used as a default. These defaults are specified by domain but will be applied to every project-domain namespace combination.
```

Comment:
Passive. Who or what uses this default?

Passive. Who or what specifies the defaults by domain?